src/providers/codebuild.ts

```diff
@@ -26,11 +26,11 @@ export interface CodeBuildRunnerProps extends RunnerProviderProps {
   readonly imageBuilder?: IImageBuilder;
 
   /**
-   * GitHub Actions label used for this provider.
+   * GitHub Actions label used for this provider. If multiple labels are specific, a workflow must specify all of them for this provider to be used.
    *
    * @default 'codebuild'
    */
-  readonly label?: string;
+  readonly label?: string | string[];
```

Comment:
shouldn`t this be plural as you`ve added in other places  "labels" as property ?

Reply:
Backwards compatibility for now. See https://github.com/CloudSnorkel/cdk-github-runners/pull/136#discussion_r1010891868